pkg/fixtures/writer_ddb.go

```diff
@@ -16,20 +16,16 @@ type dynamoDbFixtureWriter struct {
 
 func DynamoDbFixtureWriterFactory(modelId *mdl.ModelId) FixtureWriterFactory {
 	return func(cfg cfg.Config, logger mon.Logger) FixtureWriter {
-		writer := NewDynamoDbFixtureWriter(cfg, logger)
+		writer := &dynamoDbFixtureWriter{
+			config: cfg,
+			logger: logger,
+		}
 		writer.WithModelId(modelId)
```

Comment:
you can put the modelid into the object right away

Reply:
✅